indico/web/templates/layout/base.html

```diff
@@ -1,6 +1,6 @@
 {% block page %}
     <div class="{% block page_class %}management-page{% endblock %}"
-         {% if self.page_id()|trim %}id="{% filter trim %}{% block page_id %}{% endblock %}{% endfilter %}"{% endif %}>
+        {% if self.page_id()|trim %}id="{% filter trim %}{% block page_id %}{% endblock %}{% endfilter %}"{% endif %}>
```

Comment:
I'd keep it aligned with the attributes... It's semi-ugly either way but more consistent IMO